test/backends/gitlab_pages.ts

```diff
@@ -0,0 +1,82 @@
+import { expect } from 'chai';
```

Comment:
This file needs to be named `gitlab_pages.spec.ts` to be included in tests. I'm pretty sure it's not gonna pass though, unless you made those repos on Gitlab.